### PR TITLE
To-Do List: create with webpack

### PR DESCRIPTION
**Pull request by Zven94**
In this app I added:

- webpack to build the project.
- html file
- css file
- js file
- linters for better sintaxys

The app is running: it's showing a existing to-do list creating dinamic with js and a minimalistic style.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  element.innerHTML = _.arr;
+
+  element.classList.add('p-toDoList');
+
+  return element;
+}
+
+document.body.appendChild(component());
+*/
+
```
Comment (DuaneDave) [inline src/index.js]: Consider removing these comments in order to keep your codebase less cluttered up and to adhere to best practices.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const elementLabel = document.createElement('label');
+
+  elementLu.appendChild(elementLi);
+  elementLi.appendChild(elementInput);
+  elementLi.appendChild(elementLabel);
+
+  elementInput.type = 'checkbox';
+
+  elementLabel.textContent = `${element.description}, ${element.completed}, ${element.index}`;
+}
```
Comment (DuaneDave) [inline src/index.js]: Consider styling the app to look similar to the original design, if you take a look at the video, you'd quickly see some icons to the right of the interface and also the clear completed button, but your version is lacking those. I recommend including them to create the basic set up for the functionality of the app.

Your design
Screenshot


Original design
Screenshot (1)

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Zven94 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Very intuitive to implement the app using es6 classes
✔️ You rendered the to-dos in the browser correctly


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @Zven94 👋 

### Things to Appreciate
- Well Done for fixing the things requested by the previous Code Reviewer. 👍 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**